Python/Product/PythonTools/PythonTools/WorkspaceInfoBarManager.cs

```diff
@@ -128,10 +137,19 @@ class WorkspaceInfoBarManager : IVsRunningDocTableEvents, IDisposable {
                 _condaEnvCreateInfoBar.CheckAsync(),
                 _virtualEnvCreateInfoBar.CheckAsync(),
                 _packageInstallInfoBar.CheckAsync(),
-                _testFrameworkInfoBar.CheckAsync()
+                _testFrameworkInfoBar.CheckAsync(),
+                _pythonVersionNotSupportedInfoBar.CheckAsync()
             );
         }
 
+        private void TriggerPythonNotSupportedInforBar(object sender, EventArgs e) {
+            TriggerPythonNotSupportedInforBarAsync().HandleAllExceptions(_serviceProvider, GetType()).DoNotWait();
```

Comment:
Hey @huguesv isn't this a fire and forget case where we want to use   ThreadHelper.JoinableTaskFactor.runasync?

Reply:
Yes and No.

Yes, for `JoinableTaskFactory.RunAsync` but no for `ThreadHelper`. This is the case where they have mentioned we need to use our package's `JoinableTaskFactory`, not the static from `ThreadHelper`.

We do not use `JoinableTaskFactory.RunAsync` anywhere currently, we've been using `HandleAllExceptions().DoNotWait()` so far. I'd like to get our fire & forget calls changed at one point but I think we'll need a helper method on top of `RunAsync` to replicate some of the work that our helper does. I'd rather hold off for now and take the time to do it right.

Reply:
maybe HandleAllExceptions could do this